data/frame_meta.go

```diff
@@ -44,6 +50,15 @@ func (f *Frame) AppendNotices(notices ...Notice) {
 	f.Meta.Notices = append(f.Meta.Notices, notices...)
 }
 
+// QueryResultMetaStat matches https://github.com/grafana/grafana/blob/master/packages/grafana-data/src/types/data.ts#L53
+type QueryResultMetaStat struct {
```

Comment:
There are a few properties that make this confusing. Perhaps if it has to be this way, commenting might help:

- Comment doesn't specify what it is for. Can keep JS reference but should say what it is here. 
- "QueryResultMetaStat" is attached to FrameMeta, which is the property of Frame. However, in the backend pkg, this route that returns frames has a QueryDataResponse, which has responses: https://pkg.go.dev/github.com/grafana/grafana-plugin-sdk-go@v0.70.0/backend?tab=doc#QueryDataResponse . This is confusing because a QueryResultMetaStat is a member of the Frame, but the Frame is encapsulated in a QueryDataResponse from the backend package.
 - FieldConfig is a property of this, but also a property of all of a Frame Fields. When should one use which?
 - DisplayName is also a property of of the FieldConfig, so we have QueryResultMetaStat.FieldConfig.DisplayName and QueryResultMetaStat.DisplayName
 - Not sure what the value property is for, or why it is 32.

Reply:
Re: `QueryResultMetaStat`: I think maybe the confusion just comes from the name, but that's how it's named in the typescript interface so I'd be reluctant to change it.

> FieldConfig is a property of this, but also a property of all of a Frame Fields. When should one use which?
I'm not sure to be honest, but this is how it is in the typescript interface as well.

> DisplayName is also a property of of the FieldConfig, so we have QueryResultMetaStat.FieldConfig.DisplayName and QueryResultMetaStat.DisplayName
In the typescript interface I believe the property is defined again for `QueryResultMetaStat` so as to make it non-optional. Not sure if there's an equivalent way to enforce this in go.

> Not sure what the value property is for, or why it is 32.
`value` is the numeric value of some arbitrary statistic. It should be 64 bit though, which I'll correct

Reply:
@kylebrandt -- in [typescript](https://github.com/grafana/grafana/blob/master/packages/grafana-data/src/types/data.ts#L52), the structure is defined as:
```
export interface QueryResultMetaStat extends FieldConfig {
  displayName: string;
  value: number;
}
```

This means the structure is identical to FieldConfig except that `displayName` is required rather than optional and that it has a `value` property.

Field config controls how values are displayed (their units, name, etc).  This lets you configure how the stat values should get displayed.  When to use filed config vs stats?  FieldConfig is how to display each value, stats controls how the stat gets rendered in the inspect tab:

![image](https://user-images.githubusercontent.com/705951/86607041-a3d91c80-bf5d-11ea-84c2-d34b3332e7ec.png)